src/backend/distributed/commands/vacuum.c

```diff
@@ -380,10 +380,33 @@ DeparseVacuumStmtPrefix(CitusVacuumParams vacuumParams)
 
 	if (vacuumParams.index_cleanup != VACOPTVALUE_UNSPECIFIED)
 	{
-		appendStringInfoString(vacuumPrefix,
-							   vacuumParams.index_cleanup == VACOPTVALUE_ENABLED ?
-							   "INDEX_CLEANUP," : "INDEX_CLEANUP false,"
-							   );
+		switch (vacuumParams.index_cleanup)
+		{
+			case VACOPTVALUE_ENABLED:
+			{
+				appendStringInfoString(vacuumPrefix, "INDEX_CLEANUP,");
```

Comment:
I'm a little confused by this.

Why don't we send `INDEX_CLEANUP on` when `VACOPTVALUE_ENABLED`? If we don't add anything, PG on the workers would qualify that as `auto`, which is wrong?